parent/pom.xml

```diff
@@ -1641,6 +1618,28 @@
           <flattenMode>ossrh</flattenMode>
         </configuration>
       </plugin>
+
+      <!-- Google code format plugin -->
+      <plugin>
+        <groupId>com.diffplug.spotless</groupId>
+        <artifactId>spotless-maven-plugin</artifactId>
+        <configuration>
+          <java>
+            <googleJavaFormat>
+              <version>${plugin.version.google-java-format}</version>
+              <style>GOOGLE</style>
+            </googleJavaFormat>
+          </java>
+        </configuration>
+        <dependencies>
+          <dependency>
+            <groupId>com.google.googlejavaformat</groupId>
+            <artifactId>google-java-format</artifactId>
+            <version>${plugin.version.google-java-format}</version>
+          </dependency>
+        </dependencies>
+      </plugin>
+
```

Comment:
I sort of expected this to be in the bom now. Could you explain a little bit why you added it here and not in the bom?

Reply:
🤔 I guess that I worry that it's less visible to our team if the plugin is defined in more detail in the bom than in the parent pom. The way it is defined now, the bom only defines the version of the plugin and no other details. 

But, it might actually make more sense to put it in the bom. WDYT is the best?

Reply:
I'm not sure :sweat_smile:
I think I'm leaning a bit towards keeping all things related to spotless in one place, so in the bom. OTOH, most of our other plugins are configured in the parent :shrug: . I'm fine with either.

Reply:
I think I'll leave it alone for now. I'd prefer not to mess too much with the bom. We can always improve it later.